.github/workflows/mkdocs_ci.yml

```
@@ -0,0 +1,31 @@
+name: mkdocs
+on:
+  push:
+    branches:
+      - main

```

We could add the docs paths here so we only deploy when necessary.

```suggestion
    branches:
      - main
    paths: ['docs/**','.github/**','mkdocs.yml','user-requirements.txt']
```

---

Does that mean only when changes are made to those files?

---

Yes exactly, so merges with non-docs changes don't trigger a docs deploy.

---

I like it. Except for stuff that has API changes, when we're eventually auto generating that part.  

Actually @jorenn92 seems to respond more when it's on GitHub. 😂